linguicator_predictor/models/en/distilgpt2.py

```diff
@@ -0,0 +1,66 @@
+'''
```

Comment:
From PEP8: "For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in PEP 257."